src/script.js

```diff
@@ -65,35 +65,42 @@ $j(() => {
 
 	let startScreenHotkeys = {
 		KeyF: {
-			onkeydown(event) {
-				if (event.shiftKey) {
-					fullscreen.toggle();
-				}
+			keyDownTest(event) {
+				return event.shiftKey;
+			},
+			keyDownAction() {
+				fullscreen.toggle();
 			},
 		},
 		KeyL: {
-			onkeydown(event) {
-				if (event.metaKey || event.ctrlKey) {
-					readLogFromFile()
-						.then((logstr) => JSON.parse(logstr))
-						.then((log) => G.gamelog.play(log))
-						.catch((err) => {
-							alert('An error occurred while loading the log file');
-							console.log(err);
-						});
-				}
+			keyDownTest(event) {
+				return event.metaKey && event.ctrlKey;
```

Comment:
Change the key combination to CTRL + META + L. It was previously CTRL + L or META + L which seems incorrect.